docs/SNIPPET.txt

```diff
@@ -0,0 +1,8 @@
+module HelloWorld where
+
+import Prelude
+import Data.Maybe (Maybe(Just, Nothing))
```

Comment:
Ok, done. Would you eyeball this to make sure it looks right? (Having never actually written any Purescript I fear the worst).